python/pyspark/ml/tests.py

```diff
@@ -263,6 +263,23 @@ def test_ngram(self):
         transformedDF = ngram0.transform(dataset)
         self.assertEquals(transformedDF.head().output, ["a b c d", "b c d e"])
 
+    def test_stopwordsremover(self):
+        sqlContext = SQLContext(self.sc)
+        data = ["a", "panda"]
+        dataset = sqlContext.createDataFrame([
+            ([data])], ["input"])
```

Comment:
The previous test in the file was also in this style (test_ngram), should I change both or keep as is?